src/common/Constants.ts

```diff
@@ -385,6 +385,34 @@ export const EXPECTED_L1_TO_L2_MESSAGE_TIME = {
 };
 
 export const OPSTACK_CONTRACT_OVERRIDES = {
+  [CHAIN_IDs.BLAST]: {
+    l1: {
+      AddressManager: "0xE064B565Cf2A312a3e66Fe4118890583727380C0",
+      L1CrossDomainMessenger: "0x5D4472f31Bd9385709ec61305AFc749F0fA8e9d0",
+      L1StandardBridge: "0x697402166Fbf2F22E970df8a6486Ef171dbfc524",
+      StateCommitmentChain: "0x0000000000000000000000000000000000000000",
+      CanonicalTransactionChain: "0x0000000000000000000000000000000000000000",
+      BondManager: "0x0000000000000000000000000000000000000000",
+      OptimismPortal: "0x0Ec68c5B10F21EFFb74f2A5C61DFe6b08C0Db6Cb",
+      L2OutputOracle: "0x826D1B0D4111Ad9146Eb8941D7Ca2B6a44215c76",
+      OptimismPortal2: "0x0000000000000000000000000000000000000000",
+      DisputeGameFactory: "0x0000000000000000000000000000000000000000",
+    },
+    // https://github.com/blast-io/blast/blob/master/blast-optimism/op-bindings/predeploys/addresses.go
+    l2: {
+      L2CrossDomainMessenger: "0x4200000000000000000000000000000000000007",
+      L2ToL1MessagePasser: "0x4200000000000000000000000000000000000016",
+      L2StandardBridge: "0x4200000000000000000000000000000000000010",
+      OVM_L1BlockNumber: "0x4200000000000000000000000000000000000013",
+      OVM_L2ToL1MessagePasser: "0x4200000000000000000000000000000000000016",
+      OVM_DeployerWhitelist: "0x4200000000000000000000000000000000000002",
+      OVM_ETH: "0xDeadDeAddeAddEAddeadDEaDDEAdDeaDDeAD0000",
+      OVM_GasPriceOracle: "0x420000000000000000000000000000000000000F",
+      OVM_SequencerFeeVault: "0x4200000000000000000000000000000000000011",
+      WETH: "0x4300000000000000000000000000000000000004",
```

Comment:
We should probably reuse the existing WETH definition we have for this.

```suggestion
      WETH: TOKEN_SYMBOLS_MAP.WETH.addresses[CHAIN_IDs.BLAST],
```

Reply:
Fixed here: https://github.com/across-protocol/relayer/pull/1660/commits/5f6b127d63a16d20919ecef9305c27ae5da4c31d